packages/react-next/src/components/SwatchColorPicker/ColorPickerGridCell.base.tsx

```diff
@@ -0,0 +1,158 @@
+import * as React from 'react';
+import { ITheme, mergeStyleSets } from '../../Styling';
+import { classNamesFunction, memoizeFunction } from '../../Utilities';
+import { getColorFromString } from 'office-ui-fabric-react/lib/utilities/color/getColorFromString';
+import { GridCell } from 'office-ui-fabric-react/lib/utilities/grid/GridCell';
```

Comment:
You should move `utilities/grid` to react-next at the same time since it's tightly coupled to (and only used by) SwatchColorPicker.